lib/src/rules.dart

```diff
@@ -105,6 +106,7 @@ void registerLintRules() {
     ..register(new AvoidReturningNull())
     ..register(new AvoidReturnTypesOnSetters())
     ..register(new AvoidReturningThis())
+    ..register(new DoNotCreateSetterWithoutGetter())
```

Comment:
nit: I think we try to keep these lines in alphabetic order (which it was before the rename).